### PR TITLE
Added the definition for the npm package simple-url-cache

### PR DESCRIPTION
**Pull request by a-lucas**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

https://www.npmjs.com/package/simple-url-cache

### CODE REVIEW

**Comment by a-lucas** [comment]
Alright, got the Travis CI test pass ok, I'm not sure why the @dt-bot marks it as fail, also, package naming convention is good too.

Cheers, keep the good work on guys!

**Comment by vvakame** [comment]
*shelljs/shelljs.d.ts*

to author (@nikeee). Could you review this PR?
:+1: or :-1:?

Checklist

* [ ] pass the Travis CI test?

**Comment by vvakame** [comment]
FYI https://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html

File: shelljs/shelljs.d.ts [hunk 1]
```diff
@@ -511,6 +511,18 @@ declare module "shelljs"
...
 
+
+    export function touch(...files: string[]): void;
+    export function touch(files: string[]): void;
+
+    export function touch(options: string, ...files: string[]): void;
+    export function touch(options: string, files: string[]): void;
+    
+    export function touch(options: Object, ...files: string[]): void;
+    export function touch(options: Object, files: string[]): void;
```
Comment (nikeee) [inline shelljs/shelljs.d.ts]: We could be more precise concerning the options. They should be restricted to these string literals:
https://github.com/shelljs/shelljs#touchoptions-file_array

E.g.
```TypeScript
type TouchOptions = "-a" |  "-c" | "-m" | "-d" | "-r";
```

Comment (a-lucas) [inline shelljs/shelljs.d.ts]: Yes, I just pushed that, thank you.
Any recommendation on how to define empty parameters?

```typescript
interface touchOptionsArray = {
   '-a'?: null;
   '-c'?: null; 
}
```

Comment (nikeee) [inline shelljs/shelljs.d.ts]: Well, I just noticed that the options strings can be combined into one, eg `-ac`. I think the best way to model this is to just use the raw strings/objects that you proposed earlier.
Sorry for the confusion.

**Comment by nikeee** [comment]
@a-lucas Well, I just noticed that the options strings can be combined into one, eg -ac. I think the best way to model this is to just use the raw strings/objects that you proposed earlier.
Sorry for the confusion.

**Pull request merged**